standards/practices/addingReadMe.md

```diff
@@ -0,0 +1,19 @@
+
+# Proposal to add a README
```

Comment:
Can we call this "Adding READMEs" instead? It'll be more consistent with existing standards and the PR is a proposal, but once it's accepted and merged, this document will be a standard.

Can you also rename the file to `adding-readmes` for consistency?

I know these are somewhat tedious requests, but the more the standards grow, the more important consistency will become.